staging/src/k8s.io/kube-scheduler/config/v1beta1/types_pluginargs.go

```diff
@@ -77,14 +77,18 @@ type PodTopologySpreadArgs struct {
 	metav1.TypeMeta `json:",inline"`
 
 	// DefaultConstraints defines topology spread constraints to be applied to
-	// pods that don't define any in `pod.spec.topologySpreadConstraints`.
-	// `topologySpreadConstraint.labelSelectors` must be empty, as they are
-	// deduced the pods' membership to Services, Replication Controllers, Replica
-	// Sets or Stateful Sets.
-	// Empty by default.
-	// +optional
+	// Pods that don't define any in `pod.spec.topologySpreadConstraints`.
+	// `.defaultConstraints[*].labelSelectors` must be empty, as they are
+	// deduced the Pod's membership to Services, ReplicationControllers,
+	// ReplicaSets or StatefulSets.
+	// If empty, the default constraints prefer to spread Pods across Nodes and Zones
 	// +listType=atomic
```

Comment:
add back "// +optional" for this config and `DisableDefaultConstraints` below.

Reply:
What was that for again? What's the documentation for it?

I arrived to this, which doesn't help https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go#L109

Reply:
We'd better to keep the still valid comment directive here. I don't have a better documentation for it either, seems like this will help to generate a better API documentation? I am not sure.